mypyc/irbuild/specialize.py

```diff
@@ -185,6 +185,33 @@ def translate_set_from_generator_call(
     return None
 
 
+@specialize_function('builtins.min')
+def faster_min(builder: IRBuilder, expr: CallExpr, callee: RefExpr) -> Optional[Value]:
+    if (len(expr.args) > 0
+            and expr.arg_kinds == [ARG_POS, ARG_POS]):
+        x, y = builder.accept(expr.args[0]), builder.accept(expr.args[1])
+        result = Register(builder.node_type(expr))
+        comparison = builder.binary_op(x, y, '<', expr.line)
```

Comment:
Based on a quick experiment, it seems that CPython does actually evaluate `y < x` instead of `x < y` when doing `min(x, y)`. Can you double check this and update the implementation accordingly if that's the case? (This is not a big deal, but it's better to remain as close to CPython as possible.)

Reply:
Thanks for pointing out! I checked the `bltinmodule/min_max()` in CPython and it does eval `y < x` when doing `min(x, y)`. To be more specific, it evaluates arguments from last to first.